src/DateChanger.tsx

```diff
@@ -38,7 +37,6 @@ export interface DateChangerProps {
   onEdit: () => void
   onSave: (date: string) => void
   editIcon?: IconDefinition
-  saveIcon?: IconDefinition
```

Comment:
This prop is used in `src/DateChanger.stories.tsx` (the examples for the date changer component) in one of the components. This causes the type checks and build to fail at the moment...

Reply:
fixed